server/servercontext.py

```diff
@@ -70,8 +67,7 @@ def __contains__(self, connection):
         )
 
     async def _do_broadcast(self, validate_fn, send_fn, message):
-        server.stats.incr('server.broadcasts')
-
+        metrics.server_broadcasts.labels("raw").inc()
```

Comment:
And now there is both a "raw" and a "regular" broadcast function, so if we're tagging the "raw" ones we should also be tagging the "regular" ones.